shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -283,97 +178,8 @@ export default {
       return info;
     },
 
-    fakeMachines() {
```

Comment:
I haven't seen the behavior described in the comment. It appears were able to scale up and down and the machines/machine pools sync properly without all of these extra hoops.

Reply:
this is like the project/namespace list were we should show the group even if there are no rows. for instance if a pool has been scaled down to zero but we should still show the pool group. Does this still work with SSP off and on?